docs/CustomApp.md

```diff
@@ -47,39 +56,42 @@ const reducer = combineReducers({
     routing: routerReducer,
 });
 const sagaMiddleware = createSagaMiddleware();
+const history = createHistory();
 const store = createStore(reducer, undefined, compose(
-    applyMiddleware(sagaMiddleware, routerMiddleware(hashHistory)),
+    applyMiddleware(sagaMiddleware, routerMiddleware(history)),
     window.devToolsExtension ? window.devToolsExtension() : f => f,
 ));
 const restClient = simpleRestClient('http://path.to.my.api/');
 sagaMiddleware.run(crudSaga(restClient));
 
-// the resources array is used for the menu
-const resources = [
-    { name: 'posts', list: PostList },
-    { name: 'comments', list: CommentList },
-];
-
-const LayoutWithTitle = withProps({ title: 'My Admin' })(Layout);
-
-// initialize the router
-const history = syncHistoryWithStore(hashHistory, store);
-
 // bootstrap redux and the routes
 const App = () => (
     <Provider store={store}>
         <TranslationProvider messages={messages}>
-            <Router history={history}>
-                <Route path="/" component={LayoutWithTitle} resources={resources}>
-                    <IndexRoute component={Dashboard} />
-                    <CrudRoute path="posts" list={PostList} create={PostCreate} edit={PostEdit} show={PostShow} remove={Delete} />
-                    <CrudRoute path="comments" list={CommentList} create={CommentCreate} edit={CommentEdit} remove={Delete} />
-                    <CrudRoute path="users" list={UserList} create={UserCreate} edit={UserEdit} remove={Delete} />
-                </Route>
-            </Router>
+            <ConnectedRouter history={history}>
+                <MuiThemeProvider>
+                    <AppBar title="My Admin" />
+                    <Switch>
+                        <Route exact path="/" component={Dashboard} />
+                        <Route exact path="/posts" hasCreate render={(routeProps) => <PostList resource="posts" {...routeProps} />} />
```

Comment:
No, this Custom App documentation is already a corner case that we don't want to emphasize.